src/ripple/core/impl/JobQueue.cpp

```diff
@@ -168,15 +168,13 @@ JobQueue::addLoadEvents(JobType t, int count, std::chrono::milliseconds elapsed)
 bool
 JobQueue::isOverloaded()
 {
-    int count = 0;
-
     for (auto& x : m_jobData)
     {
         if (x.second.load().isOver())
-            ++count;
+            return true;
     }
 
-    return count > 0;
```

Comment:
Yup, that would work.  It would look like this:
```
    return std::any_of(
        m_jobData.begin(),
        m_jobData.end(),
        [](std::pair<const JobType, JobTypeData>& jobData) {
            return jobData.second.load().isOver();
        });
```
So I'm not convinced that using `std::any_of` actually improves readability.  Your call.

Reply:
Yep, although the quick exist is already an improvement!

Reply:
Done. I used `auto` to make it slightly less awkward to read.